### PR TITLE
Update linting and introduce stylelint

### PR DESCRIPTION
**Pull request by edg2s**

### CODE REVIEW

File: src/less/widgets/mwstew.ui.PageHeadSectionWidget.less [hunk 1]
```diff
@@ -17,8 +17,8 @@
 
 	&-title {
 		position: relative;
-		top: ~"calc(" @pageHeadSectionHeight / 2 ~" - 4em)";
-		text-shadow: rgba(0, 0, 0, 0.6) 0 1px 2px;
+		top: ~'calc(' @pageHeadSectionHeight / 2 ~' - 4em)';
```
Comment (edg2s) [inline src/less/widgets/mwstew.ui.PageHeadSectionWidget.less:20]: BTW I'm not sure what you're trying to achieve here w.r.t. to vertical alignment. Pretty sure you can do it without 'calc' though.

File: package.json [hunk 2]
```diff
@@ -9,15 +9,14 @@
     "url": "https://github.com/mooeypoo/MWStew"
   },
   "devDependencies": {
-    "grunt": "0.4.5",
-    "grunt-cli": "0.1.13",
```
Comment (edg2s) [inline package.json:13]: cli is no longer needed explicitly

**Comment by edg2s** [comment]
Rebased

**Comment by mooeypoo** [comment]
Sorry for the delay (for some reason i thought it needed a rebase, and kept postponing doing it due to lack of time, meh) -- thanks for this!!

File: src/less/widgets/mwstew.ui.PageHeadSectionWidget.less (hunk 1 above)
Comment (mooeypoo) [reply src/less/widgets/mwstew.ui.PageHeadSectionWidget.less:20]: I think I was trying to make it responsive, while getting the title to appear properly above the section. I can definitely do it with a wrapper whose width is responsive, though. I'll add that to my todo list.

**Pull request merged**